apis/core/v1alpha1/backupsession_types.go

```diff
@@ -75,10 +80,10 @@ type BackupSessionStatus struct {
 	// +optional
 	Duration string `json:"duration,omitempty"`
 
-	// Deadline specifies the deadline of backup. BackupSession will be
-	// considered Failed if backup does not complete within this deadline
+	// Deadline specifies the deadline of backup. Backup will be
+	// considered Failed if it does not complete within this deadline
 	// +optional
-	Deadline *metav1.Time `json:"sessionDeadline,omitempty"`
+	Deadline *metav1.Time `json:"backupDeadline,omitempty"`
```

Comment:
fix the json tag.

Reply:
Should it be just deadline? If yes, wouldn't it mean the deadline of backupsession?